README.md

```diff
@@ -678,6 +678,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Srinath](https://github.com/srinath1412001)
   - [Srinivas](https://github.com/Mrdecent08)
   - [Sri Venkat](https://github.com/srivenkat13)
+  - [Siddharth Singh Tanwar](https://github.com/SiddharthSinghTanwar)
```

Comment:
Please add yourself in alphabetical order.

Reply:
Sorry about that, added it correctly now.